code/game/objects/structures/window.dm

```diff
@@ -444,7 +444,7 @@
 /obj/structure/window/plasma/reinforced
 	name = "reinforced plasma window"
 	desc = "A window made out of a plasma-silicate alloy and a rod matrice. It looks hopelessly tough to break and is most likely nigh fireproof."
-	icon_state = "rplasmawindow"
+	icon_state = "plasmawindow"
```

Comment:
Who the fuck is the cuck who named them plasmarwindow. Couldn't find the right path since that breaks the convention of literally every other fucking window.

Fixing now.